feat(mc): new CLM connector (#5384)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-clm-restapi.md

---
id: applications-monitoring-centreon-clm-restapi
title: Centreon Log Management Rest API
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **Centreon Log Management Rest API** connector through the
**Configuration > Connectors > Monitoring Connectors** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

The Monitoring Connector **Centreon Log Management Rest API** brings a host template:

* **App-Monitoring-Centreon-CLM-Restapi-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Monitoring-Centreon-CLM-Restapi-custom" label="App-Monitoring-Centreon-CLM-Restapi-custom">

| Service Alias | Service Template                                                | Service Description                          |
|:--------------|:----------------------------------------------------------------|:---------------------------------------------|

> | Alert-Events  | App-Monitoring-Centreon-CLM-Alert-Events-Generic-Restapi-custom | Counts the number of alert events per status |

Reviewer: ```suggestion
| Alert-Events  | App-Monitoring-Centreon-CLM-Alert-Events-Generic-Restapi-custom | Counts the total number of current alert events, then breaks down the total by event status |
```

> The services listed above are created automatically when the **App-Monitoring-Centreon-CLM-Restapi-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias | Service Template                                             | Service Description                                                                |
|:--------------|:-------------------------------------------------------------|:-----------------------------------------------------------------------------------|

> | Log-Count     | App-Monitoring-Centreon-CLM-Log-Count-Generic-Restapi-custom | Counts the number of logs corresponding to the query ingested during the last hour |

Reviewer: ```suggestion
| Log-Count     | App-Monitoring-Centreon-CLM-Log-Count-Generic-Restapi-custom | Counts the number of logs ingested during the last hour matching a query  |
```

> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.

</TabItem>
</Tabs>

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics and statuses linked to each service.

<Tabs groupId="sync">
<TabItem value="Alert-Events" label="Alert-Events">

| Name                  | Unit  |
|:----------------------|:------|
| alerts.total.count    | count |
| alerts.unknown.count  | count |
| alerts.ok.count       | count |
| alerts.warn.count     | count |
| alerts.error.count    | count |
| alerts.critical.count | count |

</TabItem>
<TabItem value="Log-Count" label="Log-Count">

| Name      | Unit  |
|:----------|:------|
| log.count | count |

</TabItem>
</Tabs>

## Prerequisites

> Only an API token and your organization name are needed to use the plugin.

Reviewer: ```suggestion
To use this plugin, generate a token in Centreon Log Management on the **Administration > Token Management** page.
```
